README.md

```diff
@@ -1,4 +1,14 @@
 # PySlackers Community
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
As weird as it sounds, do we need a CI process to run this and auto-commit/push on merge into master?

Reply:
It doesn't sound weird at all. You're saying that once this is merged that travis runs the `npm run doctoc` command, right?

Reply:
Having Travis run it would be awesome. I'm afraid it might drive people away it they need to figure out how to run `doctoc`.